app/web/features/search/SearchBox.tsx

```diff
@@ -55,42 +55,39 @@ export default function SearchBox({
   const { t } = useTranslation([GLOBAL, SEARCH]);
   const classes = useStyles();
 
-  const { control, setValue, errors, register, handleSubmit } = useForm({
+  const { control, errors } = useForm({
     mode: "onChange",
   });
 
-  function handleOnChangeAutocomplete(event: "" | GeocodeResult) {
+  const handleOnChangeAutocomplete = (event: GeocodeResult) => {
     if (event) {
       setLocationResult(event);
     }
-  }
+  };
 
-  function handleOnChangeKeyword(event: React.ChangeEvent<HTMLInputElement>) {
+  const handleOnChangeKeyword = (
+    event: React.ChangeEvent<HTMLInputElement>
+  ) => {
     setQueryName(event.target.value);
-    setLocationResult({
-      ...locationResult,
-      location: undefined,
-    });
-  }
+    setLocationResult(undefined);
```

Comment:
same here, can't set a location to undefined (sorry for the hassle, you know, need to maintain the page behavior 😅 )

Reply:
don't worry, I will update it to make like before but with the new types

Reply:
Sure, feel free to take over this PR. Probably easier than copy pasting and merge conflicts!